diagnostic.md

```diff
@@ -6,21 +6,28 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    Inside the application router, you will define to Ember the different routes
+    you need in order to display certain content. i.e:
+    Router.map(function () {
+       this.route('about');
+     });
+
+     In the route, you will define a model which Ember will interact with to
+     fetch data.
```

Comment:
Yes, the Ember Application Router defines the route, and the Ember Route specifies how to load model data using the model hook.